pandas/core/frame.py

```diff
@@ -9306,8 +9306,8 @@ def _AXIS_NAMES(self) -> Dict[int, str]:
 ops.add_special_arithmetic_methods(DataFrame)
 
 
-def _from_nested_dict(data):
-    new_data = collections.defaultdict(dict)
+def _from_nested_dict(data) -> collections.defaultdict:
+    new_data: collections.defaultdict = collections.defaultdict(dict)
```

Comment:
you *should*n't have to type this

Reply:
i agree, but mypy complains when it isnt annotated